src/components/include/utils/threads/thread.h

```diff
@@ -137,12 +132,12 @@ class Thread {
   /**
    * \brief Used to request actions from worker thread.
    */
-  ThreadCommand thread_command_;
+  volatile ThreadCommand thread_command_;
```

Comment:
@LuxoftAKutsan, the intention is not to make it thread safe but suppress optimizations. Just to ensure that all threads read actual values in RAM.